rosdep/base.yaml

```diff
@@ -2239,6 +2239,10 @@ libncurses-dev:
   gentoo: [sys-libs/ncurses]
   macports: [ncurses]
   ubuntu: [libncurses5-dev]
+libneon27-gnutls-dev:
+  debian: [libneon27-gnutls-dev]
+  gentoo: [neon]
```

Comment:
The gentoo package should list the category: `net-libs/neon`

Reply:
ok I did